charts/trento-server/Chart.yaml

```diff
@@ -7,7 +7,7 @@ description: The trento server chart contains all the components necessary to ru
 type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates
-version: 2.3.1-dev
+version: 2.3.1-dev1
```

Comment:
@rtorrero @arbulu89 asking for confirmation.

Should I change also 
```
#!BuildTag: trento/trento-server:2.3.0
#!BuildTag: trento/trento-server:2.3.0-build%RELEASE%
```
to
```
#!BuildTag: trento/trento-server:2.3.1-dev1
#!BuildTag: trento/trento-server:2.3.1-dev1-build%RELEASE%
```
?

Reply:
Where do you want to change this?
In the web obs container Dockerfile?
I wouldn't. I don't think anyone uses OBS dev containers.
Besides that, i don't think OBS follows real semver versioning semantics, so having `-` there wouldn't mean that this version is "smaller" than 2.3.1, which could cause some issue.

Reply:
Just asking to be sure I wasn't missing anything :see_no_evil: 
Thanks for the answer